HISTORY.md

```diff
@@ -4,7 +4,7 @@ In the beginning, programmers created the databases. Now the databases were form
 
 And so [Salesforce.org](http://salesforce.org/) said “Let there be data,” and there was Snowfakery. And it was good.
 
-## Snowfakery 1.8
+## Snowfakery 1.8 and 1.8.1
```

Comment:
I know I wouldn't appreciate a project that does this. Add a new heading with a release note like "fixed packaging issue" or something like that.